properties.go

```diff
@@ -622,6 +622,30 @@ func (p *Properties) WriteComment(w io.Writer, prefix string, enc Encoding) (n i
 	return
 }
 
+// ToMap returns a copy of the properties as a map
```

Comment:
Maybe you can do a quick dig through the stdlib to find some precedence. I don't think they have `ToXXX` functions there. `Map` could work but might be too generic.